docs/private-networks/reference/api/index.md

```diff
@@ -2075,6 +2075,67 @@ curl -X POST --data '{"jsonrpc": "2.0","method": "priv_newFilter","params": ["4r
 }
 ```
 
+</TabItem>
+</Tabs>
+
+### `priv_traceTransaction`
+
+Provides a [transaction trace](../../../public-networks/reference/api#trace_transaction) for a private transaction. 
+
+#### Parameters
+
+- `privacyGroupId`: _string_ - the privacy group ID associated with the transaction
+
+- `transactionHash`: _string_ - the hash of the private transaction to trace
+
+#### Returns
+
+`result`: _array_ of _objects_ - list of [calls to other contracts](../../../public-networks/reference/trace-types.md#trace) containing one object per call, in the order called by the transaction. 
+If revert reason is enabled with [`--revert-reason-enabled`](../../../public-networks/reference/cli/options.md#revert-reason-enabled), the returned list items include the [revert reason](../../how-to/send-transactions/revert-reason.md).
+
+<Tabs>
+  
+<TabItem value="curl HTTP request" label="curl HTTP request" default>
+
+```bash
+curl -X POST --data '{"jsonrpc": "2.0", "method": "priv_traceTransaction", "params": ["0xf3ab9693ad92e277bf785e1772f29fb1864904bbbe87b0470455ddb082caab9d"], "id":1}' http://127.0.0.1:8545
+```
+
+</TabItem>
+
+<TabItem value="wscat WS request" label="wscat WS request">
+
+```bash
+{"jsonrpc":"2.0", "method": "priv_traceTransaction", "params": ["0xf3ab9693ad92e277bf785e1772f29fb1864904bbbe87b0470455ddb082caab9d"], "id":1}
+```
+
+</TabItem>
+
+<TabItem value="JSON result" label="JSON result">
+
+```json
```

Comment:
result will look like this - https://besu.hyperledger.org/development/public-networks/reference/api#trace_transaction

Reply:
Refer to trace_transaction api

Reply:
@NickSneo  I used that method to create the docs. The only change I applied was changing it to `private_traceTransaction` instead of `trace_transaction`. Could you please review again and/or provide a suggestion of exactly what should be in there?  I think this is ready to go, just need approval.

Reply:
Sorry @NickSneo PTAL and let me know if this can be approved. Thanks a million

Reply:
Thank you, LGTM